### DISCUSSION TITLE
German Translation Lane - Fahrspur to Fahrstreifen

### DISCUSSION THREAD

**Discussion by puerki** [osmandapp/OsmAnd#11243] [General]
Hello,
is it possible to change the translation of „Lane“ in German from „Fahrspur“ to „Fahrstreifen“?

Because in the ÖNORM V5050 is the Definition (and in the RVS, STVO):

3.57 Fahrspur:
Auf die Fahrfläche übertragene Abrollspur von Fahrzeugreifen.

3.58 Fahrstreifen:
Ein Teil der Fahrbahn, dessen Breite für die Fortbewegung einer Reihe mehrspuriger Fahrzeuge ausreicht.

also on wikipedia
https://de.wikipedia.org/wiki/Straßenquerschnitt#Fahrstreifen
and the German STVO
https://www.stvo.de/strassenverkehrsord ... tfahrzeuge

Thanks

**Comment by vshcherb** [comment]
All translations are hosted here - https://hosted.weblate.org/projects/osmand/ but it's good to have conversation here between the authors of current translation. @sonora @scaidermern

**Comment by sonora** [comment]
I am aware that expression is used in the official norm, but to me personally it sounds rather clumsy and bureaucratic.

I am certainly not entirely against making the change, though, if that idea finds supporters in this thread.

Technically: We have 6 occurrences of the word "Fahrspur" in desciptive text and the menu, including two in the combination  "Fahrspurassistent", where the replacement may be even clumsier, not sure? Is the idea to replace in all 6? No occurrence in our voice prompts.

**Reply by pebogufi** [reply]
I also think that Fahrspur is commonly used, rather than Fahrstreifen, which is very uncommon.
So please don't change that.

**Reply by scaidermern** [reply]
I fully agree with @sonora and @pebogufi. While "Fahrstreifen" might be the correct term according to German law, most people don't use it. We should stick to "Fahrspur" instead. In the end, text and voice announcements must be understandable to the user and shouldn't sound too alien.
